.bumpversion.cfg

```diff
@@ -1,5 +1,5 @@
 [bumpversion]
-current_version = 3.0.0
+current_version = 3.0.1
```

Comment:
This probably should be 3.1.0 right as we've added a new (albeit very small) feature to the codebase?

Reply:
I'll update it to 3.1.0. Thanks.